access/types/forms.py

```diff
@@ -452,6 +458,12 @@ def grade_field(self, i, configuration):
         else:
             raise ConfigError("Unknown field type for grading: %s" % (t))
 
+        points = configuration.get('points', 0)
+        points_got = points if ok else 0
+        if configuration.get('partial_points'):
+            points_got = max(int(points*ok),0)
```

Comment:
Add spaces around the multiplication operator ` * `.

Reply:
It is confusing that `ok` sounds like a boolean variable, but you use it like a number in a multiplication. What is the type of `ok`?